src/main/java/com/ticketbooking/dto/ReservationDto.java

```diff
@@ -0,0 +1,55 @@
+package com.ticketbooking.dto;
+
+import java.util.List;
+
+public class ReservationDto {
+
+    private Long screeningId;
+    private String name;
+    private String surname;
+    private String mail;
+    private List<TicketDto> tickets;
+
+    public ReservationDto() {
```

Comment:
Czy to jest potrzebne?

Reply:
Nie, Ten konstruktor jest on domyślny i tutaj jest zbędny.